src/azure-cli-core/setup.py

```diff
@@ -66,8 +66,9 @@
     'pyyaml~=5.2',
     'requests~=2.20',
     'six~=1.12',
-    'wheel==0.30.0',
+    'wheel>=0.30.0',
```

Comment:
How about here:
https://github.com/Azure/azure-cli/blob/3b0c3248b71330b9d47f5f40732065cd8867dc48/requirements.txt#L3

Reply:
But this can not resulve [user's request about using wheel==0.34.2 ](https://github.com/Azure/azure-cli/pull/12583)

Reply:
Oh, I get it, will update.